crates/formality-prove/src/decls.rs

```diff
@@ -141,13 +143,23 @@ pub struct NegImplDeclBoundData {
 }
 
 #[term]
+#[customize(debug)]
 #[derive(Default)]
 pub enum Safety {
     #[default]
     Safe,
     Unsafe,
 }
 
+impl fmt::Debug for Safety {
```

Comment:
Isn't this the default behavior that `#[term]` would generate?

Reply:
Oh I didn't know this, or test that it would. I'll try and open a PR to clean this up.

Reply:
I've fixed this in #156